lua/apisix.lua

```diff
@@ -44,6 +45,14 @@ end
 
 
 function _M.http_init_worker()
+    local we = require("resty.worker.events")
+    local ok, err = we.configure({shm = "worker-events", interval = 0.1})
```

Comment:
where we use this worker events?

Reply:
code example: https://github.com/Kong/lua-resty-healthcheck#synopsis

I think you should know how this mini case works.